src/LanguageManager/LanguageEditView.php

```diff
@@ -0,0 +1,229 @@
+	{
+		$this->languages = $languages;
+		$this->labels    = $labels;
+	}
+
+	/**
+	 * @param int $language_id
+	 *
+	 * @return void
+	 */
+	public function render( int $language_id )
+	{
+		$language = $this->languages->get_language_by( LanguagesTable::COLUMN_ID, $language_id );
+
+		print $this->print_language_id( $language );
+		print '<table>';
+		$this->print_text_fields( $language );
+		$this->print_priority( $language );
+		$this->print_rtl( $language );
+		$this->print_site_selector( $language );
+		print '</table>';
+
+		if ( ! wp_doing_ajax() ) {
+			submit_button();
+		}
+	}
+
+	/**
+	 * Language ID in a hidden input
+	 *
+	 * @param \Inpsyde\MultilingualPress\Common\Type\Language $language
+	 *
+	 * @return void
+	 */
+	private function print_language_id( Language $language )
+	{
+		?>
+		<input type="hidden" name="language_id" value="<?=esc_attr($language->name( LanguagesTable::COLUMN_ID ) )?>">
+		<?php
+	}
+
+	/**
+	 * General text input fields
+	 *
+	 * @param Language $language
+	 *
+	 * @return void
+	 */
+	private function print_text_fields( Language $language )
+	{
+		$text_fields = [
+			LanguagesTable::COLUMN_NATIVE_NAME,
+			LanguagesTable::COLUMN_ENGLISH_NAME,
+			LanguagesTable::COLUMN_HTTP_CODE,
+			LanguagesTable::COLUMN_ISO_639_1_CODE,
+			LanguagesTable::COLUMN_LOCALE,
+		];
+
+		foreach ( $text_fields as $field ) {
+			$label = $this->labels->label( $field );
+			$value = $language->name( $field );
+			?>
+			<tr>
+				<td class="alignright">
+					<label for="<?= esc_attr( $field ) ?>_id">
+						<?= esc_html( $label ) ?>
+					</label>
+				</td>
+				<td>
+					<input
+						type="text"
+						name="<?= esc_attr( $field ) ?>"
+						id="<?= esc_attr( $field ) ?>_id"
+						value="<?= esc_attr( $value ) ?>"
+					/>
+				</td>
+			</tr>
+			<?php
+		}
+	}
+
+	/**
+	 * HTTP priority numeric input
+	 *
+	 * @param Language $language
+	 *
+	 * @return void
+	 */
+	private function print_priority( Language $language )
+	{
+		?>
+		<tr>
+			<td class="alignright">
+				<label for="<?=esc_attr( LanguagesTable::COLUMN_PRIORITY )?>_id">
+					<?=esc_attr( $this->labels->label( LanguagesTable::COLUMN_PRIORITY ) )?>
+				</label>
+			</td>
+			<td>
+				<input
+					type="number" min="1" max="10"
+					name="<?=esc_attr( LanguagesTable::COLUMN_PRIORITY )?>"
+					id="<?=esc_attr( LanguagesTable::COLUMN_PRIORITY )?>_id"
+					value="<?=esc_html( $language->name( LanguagesTable::COLUMN_PRIORITY ) )?>"
+				/>
+			</td>
+		</tr>
+		<?php
+	}
+
+	/**
+	 * Checkbox for RTL property of a language
+	 *
+	 * @param Language $language
+	 *
+	 * @return void
+	 */
+	private function print_rtl( Language $language )
+	{
+		?>
+		<tr>
+			<td></td>
+			<td>
+				<input
+					type="checkbox"
+					name="<?=esc_attr( LanguagesTable::COLUMN_RTL )?>"
+					id="<?=esc_attr( LanguagesTable::COLUMN_RTL )?>_id"
+					value="1"
+					<?=checked( true, $language->is_rtl(), false )?>
```

Comment:
See how using short echo tags can make things ugly? 😜 It's just `<?php checked( $language->is_rtl ); ?>`.